Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -51,13 +51,6 @@
     sprite: Clothing/OuterClothing/Coats/hos_trenchcoat.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Coats/hos_trenchcoat.rsi
-  - type: Armor
-    modifiers:
-      coefficients:
-        Blunt: 0.7
-        Slash: 0.7
-        Piercing: 0.4
-        Heat: 0.7
```

Comment:
Restore these and rename it Armored Trenchcoat

Reply:
I don’t think it’ll help anything to do that in all honesty. Most people who don’t know it’s armies aren’t actively inspecting it.

Reply:
you could change the sprite to visually show a little vest like det/wardens vest so you dont need to strip and examine to see its armoured

Reply:
I've mentioned it in the PR (I think) that I would prefer that solution, but I don't really sprite stuff so this was the next best option, and one that was suggested by a few people.

Reply:
> I don’t think it’ll help anything to do that in all honesty. Most people who don’t know it’s armored aren’t actively inspecting it- if they were they could see the armor values already.

This isn't an issue in SS13. If people see the HoS they assume 100% everything they have is armored.

Reply:
> This isn't an issue in SS13. If people see the HoS they assume 100% everything they have is armored.

I'll admit I haven't played SS13- but just because something works a certain way there doesn't mean it should work like that here. Evidently people aren't assuming everything the HoS has is armored, and I think making a change to better convey that idea isn't a bad thing, while also moving away from the "regular piece of clothing = armor" thing others have brought up.

Reply:
Having a "well it always worked this way" can be both a good and bad mindset. It's bad if you use it as an excuse to prevent change but it's also good to look at the history and if it needs to be improved.

In this case I don't think it needs to have a major overhaul. 

Department head fashion is also somewhat important to people. In SS13 some HoS like wearing the parade jacket, where others wear the armor, winter coat, or the trench. All have the same armor values. Same thing with beret vs HoS hat vs helmet. Doesn't stop at HoS but also extends to other head roles as well like Captain.

So making a change like how it originally is in the PR isn't great. People need to learn the game sometimes and this is an example. Now adding (Armored) to the Trenchcoat is a fine middle ground because it gives that extra hint this is an armored piece of clothing.

Reply:
Going to preface this by saying I'm probably just going to change the name to say it's armored if this sits for another day- this PR's controversial enough (even if I do think some people are just sad they're losing a trenchcoat), and as Emo said, I'll get a different answer depending entirely on which maintainer is asked. Still think there's a few things to say, though, so I'll get those out.

> An overall better approach would also to expand the examine menu.
I agree. There are several better approaches (including changing the sprite), I just did this because it's what I was capable of doing and someone suggested it. 

> Department head fashion is also somewhat important to people. In SS13 some HoS like wearing the parade jacket, where others wear the armor, winter coat, or the trench. All have the same armor values. Same thing with beret vs HoS hat vs helmet. Doesn't stop at HoS but also extends to other head roles as well like Captain.
I get that too. This PR started with me giving the HoS's winter coat armor because I was upset it looked good but was an objectively "worse" option. Thing is, even if suit variants did work like that in SS13- it doesn't _anywhere_ in 14, and some people still don't care. (I realized that's a little confusing- I'm just saying the choice between "looking good" or "good stats" already exists a lot, and we see both options being taken.) The head of security's beret and hat provide no protection compared to a helmet, people still wear it. Last game I saw a Captain wearing their winter coat instead of the carapace. The warden's armored winter coat is probably objectively worse than the vest, but people still wear that. Not like we have to say "you must look bad if you have armor on", but as-is if the Captain thinks their vest looks good and wants to wear it, they can, they just know they'll be sacrificing their armor as a result. Making the HoS do something similar isn't the end of the world. 

I probably had more to add but I've already managed to make this one hell of a comment-filled PRs. If we want to stick with the "head fashion" idea, we should probably go and make the winter coats a legitimate option too. As-is the two heads with armor only have one set and a few other less valuable clothes anyways.

Reply:
As it's been a day, I've just changed it to say it's armored.